crates/timers/tests/web.rs

```diff
@@ -68,48 +66,38 @@ fn timeout_future_cancel() -> impl Future<Item = (), Error = wasm_bindgen::JsVal
         let cell = cell.clone();
         move |_| {
             cell.set(true);
-            2
+            2 as u32
```

Comment:
Should be `2u32`

Reply:
I think the type system can infer this now. However, I can be explicit if you prefer.